lib/cloud_controller/dependency_locator.rb

```diff
@@ -201,6 +201,10 @@ def paginated_collection_renderer
       create_paginated_collection_renderer
     end
 
+    def object_collection_renderer
```

Comment:
Love this.  Can't wait to expose it in the CLI.  I was going to try to
accomplish this via a plugin but happy to just hit CC endpoints!

Greg Oehmen
Cloud Foundry Product Manager
415.205.6596


On Thu, May 14, 2015 at 4:43 PM, Joseph Palermo <notifications@github.com>
wrote:

> In lib/cloud_controller/dependency_locator.rb
> <https://github.com/cloudfoundry/cloud_controller_ng/pull/378#discussion_r30374659>
> :
>
> > @@ -201,6 +201,10 @@ def paginated_collection_renderer
> >        create_paginated_collection_renderer
> >      end
> >
> > +    def object_collection_renderer
>
> Has this method been added for a reason? We're only seeing it used in
> tests.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cloudfoundry/cloud_controller_ng/pull/378/files#r30374659>
> .
>